config/production.yaml.example

```diff
@@ -293,6 +293,30 @@ tracker:
   private: true
   # Reject peers that do a lot of announces (could improve privacy of TCP/UDP peers)
   reject_too_many_announces: false
+  
+customize_torrents:
+  # If additional trackers is more than 0, 
+  # then as many as additional trackers entries from the trakcers list will be 
+  # added to the torrent file
+  # entries are randomly chosen if number is less than whole list otherwise all
+  # 0 , empty or absense of this property means disabled   
+  # additional_trackers: 6
```

Comment:
I find this too complex. I prefer to remove the max count config and just put inside the torrent all additional trackers defined in the config.